### PR TITLE
Include Package Data?

### PR DESCRIPTION
**Pull request by Cadair**

### CODE REVIEW

File: {{ cookiecutter.package_name }}/setup.cfg [hunk 1]
```diff
@@ -14,6 +14,7 @@ packages = find:
 python_requires = >={{ cookiecutter.minimum_python_version }}
 setup_requires = setuptools_scm
 install_requires = {{ cookiecutter._install_requires }}
+include_package_data = True
```
Comment (astrofrog) [inline {{ cookiecutter.package_name }}/setup.cfg:17]: We shouldn't do this if we also provide package_data below - it should be one or the other

Comment (Cadair) [reply {{ cookiecutter.package_name }}/setup.cfg:17]: oh, I had to add this to make it work with the `package_data` thing below. Did I miss a trick?

Comment (astrofrog) [reply {{ cookiecutter.package_name }}/setup.cfg:17]: The ``include_package_data`` actually ignores ``package_data`` and uses the manifest instead as far as I understand

Comment (Cadair) [reply {{ cookiecutter.package_name }}/setup.cfg:17]: maybe I just messed up the `package_data` thing :thinking:

**Pull request closed without merging**